apps/smart-contracts/token/test/Vesting.test.ts

```diff
@@ -1256,5 +1256,95 @@ describe('Vesting', () => {
         ).revertedWith('Sender blocked')
       })
     })
+
+    describe('if user claims and is not allowed source', () => {
+      beforeEach(async () => {
+        disallowedSourceUser = unblockedUser1
+        expect(await sourceAllowlist.isIncluded(disallowedSourceUser.address)).to.eq(false)
+        const disallowedSourceUserBalanceBefore = await ppo.balanceOf(disallowedSourceUser.address)
+        const vestingBalanceBefore = await ppo.balanceOf(vesting.address)
+        const disallowedSourceUserTotalClaimedBefore = await vesting.getClaimedAmount(
+          disallowedSourceUser.address
+        )
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        await utils.setNextTimestamp(ethers.provider as any, timeAfterVestingStarted)
+        // Claim by disallowed source user
+        await vesting.connect(disallowedSourceUser).claim()
+        const disallowedSourceUserTotalClaimedAfter = await vesting.getClaimedAmount(
+          disallowedSourceUser.address
+        )
+        const expectedChangeInBalance = disallowedSourceUserTotalClaimedAfter.sub(
+          disallowedSourceUserTotalClaimedBefore
+        )
+        expect(expectedChangeInBalance).to.not.eq(0)
```

Comment:
```suggestion
        expect(expectedChangeInBalance).to.not.eq(0)
```
Same thing here, lets ensure the change in balance is correct and not just zero, I'd suggest moving this calculation to a separate function and just calling it whenever you need to verify a claim amount.